fastods/src/main/java/com/github/jferard/fastods/TableAppender.java

```diff
@@ -110,7 +110,6 @@ public void appendPreamble(final XMLUtil util, final Appendable appendable) thro
             }
         }
         appendable.append(">");
-        this.appendForms(util, appendable);
```

Comment:
I would prefer that:
1. you create a `List<XMLConvertible>forms = new ArrayList<XMLConvertible>()` field;
2. you check if this list is empty (it will be unless someone implements the forms!). If it is not, then call `appendForms`.

Reply:
yea that should be fine, i'm not sure where to add the field though?

Reply:
Good question! Let's make it simple for the moment: add it to the `TableAppender` class. Maybe we'll need to move it later, but that field will just be a placeholder for a future feature.

Reply:
Yea sounds great, would you like to add this? :)

Reply:
If you want to write it, help yourself. Otherwise I'll do it.

Reply:
I'd be happy if you could do it, shouldnt take you long, as you already have a good idea how it schould look like, thanks! :)

Reply:
No problem, I'll do it.